bin/git-rename-remote

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+old=$1
+new=$2
+
+test -z $old && echo "old remote name required." 1>&2 && exit 1
+test -z $new && echo "new remote name required." 1>&2 && exit 1
+
+if git config --get "remote.$new.fetch" > /dev/null; then
+  git remote remove $new
```

Comment:
One little thing: maybe we should also check if the `old` remote exists?
Consider the situation below:

0. There are two remotes: `own` and `upstream`.
1. Alice wants to rename `own` to `upstream`.
2. She run `git rename oen upstream`.
3. Because the typo, the operation fails.
4. However, `upstream` is removed by the this line: `git remote remove $new`.